opal/mca/btl/openib/btl_openib_component.c

```diff
@@ -2626,7 +2702,7 @@ btl_openib_component_init(int *num_btl_modules,
     struct dev_distance *dev_sorted;
     float distance;
     int index;
-    bool found;
+    bool found, disallowed_ib = false;
```

Comment:
Nit: my fossilized brain finds positive logic easier to process than negative logic.  Plus, the MCA param is `allow_ib` (not `disallow_ib`).  It might be 3% simpler to make this variable `allow_ib` to a) match the MCA param and b) make all the comparisons be positive-based logic.

But that's just my $0.02 -- feel free to disregard!